.gitattributes

```diff
@@ -1 +1,3 @@
-**/artifacts/* binary
\ No newline at end of file
+**/artifacts/* binary
+**/schema/*.json linguist-generated=true
+**/docs/book linguist-generated=true
```

Comment:
this is also in gitignore so probably not needed here?

Reply:
Yeah, thanks, removed it!